templates/contrib/dg_maps/dg-maps-map-region.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set region_classes = [
+'map-region',
+'map-region--' ~ name,
+] %}
+
+{% if settings.collapsed %}
+  {% set region_classes = region_classes|merge(['map-region--collapsible']) %}
+{% endif %}
+
+{% set hide_on_mobile = settings.hide_on_mobile ? ' hide-on-mobile' : '' %}
+
+{% if settings.collapsed %}
+  <button type="button" class="map-region-button {{ hide_on_mobile }}" data-toggle-region="{{ name }}" title="{{ "Toggle @name region"|t({ '@name': name }) }}">
```

Comment:
Is this a toggle button?
Would suggest using aria-expanded or aria-pressed.
Only if you're certain that you can manage the states.